### PR TITLE
roonbridge 1.8,100801125

### PR DESCRIPTION
**Pull request by daeho-ro**
- https://github.com/Homebrew/homebrew-cask/issues/171006

----

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/r/roonbridge.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+  sha256 "003dcaf79a3ae4bc06f62e5edd35bb40ccf32b05df28bf59182eacb9ac35ba28"
 
-  url "https://download.roonlabs.net/builds/RoonBridge.dmg",
-      verified: "download.roonlabs.net/builds/"
+  url "https://download.roonlabs.net/updates/stable/RoonBridge_#{version}.dmg",
+      verified: "download.roonlabs.net/updates/"
   name "Roon Bridge"
   desc "Music player network extender"
-  homepage "https://roonlabs.com/"
+  homepage "https://roon.app/"
```
Comment (khipp) [inline Casks/r/roonbridge.rb:9]:
```suggestion
  homepage "https://roon.app/en/"
```

The current homepage redirects to `https://roon.app/en/`.

Comment (daeho-ro) [reply Casks/r/roonbridge.rb:9]: I think it is based on the locale, my case it is going to /ko. Should I set to the default to /en?

File: Casks/r/roonbridge.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url "https://updates.roonlabs.net/update/?v=2&platform=macosx&version=&product=RoonBridge&branding=roon&branch=production&curbranch=production"
+    regex(/^machineversion=(\d+)$/i)
```
Comment (khipp) [inline Casks/r/roonbridge.rb:13]:
```suggestion
    regex(/machineversion=(\d+).*?displayversion=v?(\d+(?:\.\d+)+)/im)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[1]},#{match[0]}" }
    end
```

File: Casks/r/roonbridge.rb [hunk 3]
```diff
@@ -1,16 +1,16 @@
 cask "roonbridge" do
-  version "1.0"
-  sha256 :no_check
+  version "100801125"
```
Comment (khipp) [inline Casks/r/roonbridge.rb:2]:
```suggestion
  version "1.8,100801125"
```

File: Casks/r/roonbridge.rb [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+  sha256 "003dcaf79a3ae4bc06f62e5edd35bb40ccf32b05df28bf59182eacb9ac35ba28"
 
-  url "https://download.roonlabs.net/builds/RoonBridge.dmg",
-      verified: "download.roonlabs.net/builds/"
+  url "https://download.roonlabs.net/updates/stable/RoonBridge_#{version}.dmg",
```
Comment (khipp) [inline Casks/r/roonbridge.rb:5]:
```suggestion
  url "https://download.roonlabs.net/updates/stable/RoonBridge_#{version.csv.second}.dmg",
```

File: Casks/r/roonbridge.rb (hunk 1 above)
Comment (khipp) [reply Casks/r/roonbridge.rb:9]: Thank you, let’s keep it as is then.

**Review by khipp — APPROVED** [review]
Thank you, @daeho-ro!

**Pull request merged**